tools/pipelines/build-build-tools.yml

```diff
@@ -99,6 +107,7 @@ extends:
     - test:copyresults
     taskLint: true
     taskLintName: ci:eslint
+    poolBuild: NewLarge-linux-1ES
     poolBuild: Large
```

Comment:
Duplicate argument. Question, have you installed the Azure Pipelines VSCode extension? I'd expect that with some recent changes I made, this should have been flagged as a syntax error by VSCode.

Reply:
I have not, just installed the extension and syntax highlighting is working now and caught that issue. (Also, I have now fixed this duplicate pool definition issue)